### ISSUE TITLE
Pumping into a quarry causes the quarry to pause

### ISSUE THREAD

**Issue by Spauldo** [BuildCraft/BuildCraft#4402]
I placed a quarry about 15 blocks below the surface.  It cleared out the frame area and was working fine.

It encountered some Thermal Expansion crude oil, which it refused to dig through (treating it like lava).  It left the liquid-topped platform that anyone who's tried to quarry the nether is familiar with.  I decided to pump out the T.E. crude oil with a Buildcraft pump.

The pump worked fine, but while it was working the quarry completely stopped.  Whenever the pump would suck down a complete level and lower itself, the quarry would start again briefly.

It's not really a problem, I just thought it was a bit strange.

BuildCraft version: 7.99.23
Forge version: 14.23.5.2779
Link to crash report or log: N/A
Singleplayer or multiplayer: Single Player
Steps to reproduce: Place a quarry somewhere with liquid, run it, place a pump above it, start pump.
Additional information: I like the little hammer on the quarry arm.  It's needed something like that for a long time.

**Comment by Dame-E-in** [comment]
#4383

**Comment by Dame-E-in** [comment]
Also, the crude oil thing should be right. As long as it's dense enough.
![image](https://user-images.githubusercontent.com/20918683/56855874-67ec0580-691d-11e9-9ef1-1aee40ec218b.png)

**Comment by Spauldo** [comment]
The issue you linked very well might be related.  The main difference I see is that that issue is with the quarry's laser and lava/snow interaction, while mine is the regular mining arm and the Buildcraft pump.  I did notice that when the quarry did unpause, it ran at lightning speed for a short bit, so that part is the same.

Regarding the crude oil, I kinda suspected something like that which is why I didn't open an issue for it.  It's good to know that's the intended behavior.

**Comment by Dame-E-in** [comment]
I don't believe the snow actually has anything to do with it. Lava breaks snow. He used lava to do that. The issue arrived when Marco's quarry encountered the lava. The quarry should have just skipped over the lava but instead, the quarry had done brokeded itself visually and sonically (kind of but not really) until the lava was removed. Yours had as much to do with a hand removing a bucket of lava as a pump removing a Y level of crude oil.
 
For the laser and mining arm part, I think that functions the same, but is just textured differently. I may be an idiot thinking that but, meh XD.
